build/pubspec.yaml

```diff
@@ -18,3 +18,9 @@ dev_dependencies:
   glob: ^1.1.0
   test: ^0.12.0
   transformer_test: ^0.2.0
+
+dependency_overrides:
```

Comment:
Actually, I was planning on leaving these in until the next release. Otherwise no tests will pass etc.